tests/integration/components/webrtc-troubleshooter/component-test.js

```diff
@@ -1,19 +1,94 @@
+import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 
+const intl = Ember.Service.extend({
+  exists () {
+    return true;
+  },
+  lookup () {
+    return this.t(...arguments);
+  },
+  t (key) {
+    if (key === 'chat.presence.outOfOffice') {
```

Comment:
this line is pointless only need to mockout t function so I will remove this condition